### ISSUE TITLE
Ensure all view state is saved/restored

### ISSUE THREAD

**Issue by cryos** [OpenChemistry/tomviz#1554]
The axes grid on the view still needs to be added to the state file, and I suspect some other elements may be missing.

**Comment by mathturtle** [comment]
Superseded by #1590 and #1591?

**Comment by cryos** [comment]
No, this becomes a way to track that the remaining items in the simplified/updated menu is saved. We should probably do that in the same pull requests, but some testing and validation that everything is saved/restored should happen before we close this issue - i.e. the menu reflects the restored state, and the view reflects it too.

**Comment by willlllla** [comment]
Hi, I'm Willa, a newcomer to prof Hodven's lab. I believe these following elements are not saved after "save state" in Tomviz. I'm using Tomviz 1.3.1 on MaxOS.

In contour rendering, the surface rendering setting is not saved: whether I want to render it as surface, wireframes or points, according to a drop-down menu.

In volume rendering, the transfer mode is always set to "scalar," and works done in "scalar- gradient 1D" and "scalar- gradient 2D" are not saved. When I'm using gradient 2D, to color my object right, I have to manually adjust the position of my intensity selection box each time I load state.

**Comment by cryos** [comment]
The transfer function is being tracked in #1302, we will get to that soon, and can track gradient opacity there too. I wasn't aware of the contour issue, we should ensure that is restored. This issue is primarily concerned with ensuring the view menu state is saved/restored. To avoid overloading this issue too much let's keep visualization module state in a separate issue(s).

**Comment by cryos** [comment]
@mathturtle are all items mentioned by @willlllla addressed, I know 2D transfer should be improved now, but am not aware of patches/attention on other elements. Are all view properties now being saved that are exposed in the view menu too?

**Comment by mathturtle** [comment]
The view properties that are exposed in the view menu are all being saved.  The volume transfer mode was fixed as part of the 2D transfer functions branch.

I just tried the contour one and it is not fixed.  I also got the "Initial contour value" dialog when loading the state file, which is probably not what we want.

**Comment by mathturtle** [comment]
Ok, all of the mentioned issues are addressed now.
